src/main/java/com/powsybl/openloadflow/DefaultAcOuterLoopConfig.java

```diff
@@ -36,6 +36,8 @@ public List<AcOuterLoop> configure(LoadFlowParameters parameters, OpenLoadFlowPa
         createTransformerReactivePowerControlOuterLoop(parametersExt).ifPresent(outerLoops::add);
         // shunt compensator voltage control
         createShuntVoltageControlOuterLoop(parameters, parametersExt).ifPresent(outerLoops::add);
+        // AC emulation
+        createAcEmulationOuterLoop(parameters).ifPresent(outerLoops::add);
         // automation system
```

Comment:
I'm wondering where this loop should be. It seems to me that it is better placed right after the dsitributed slack as the flow of active current is what would mostly drive the angles. And also that any voltage control computation made with a wrong direction of power flow in an HVDC is of little.  @geofjamg  any thought on this topic ?

Reply:
Yes, I agree.

Reply:
Done